index.js

```diff
@@ -38,7 +37,8 @@ module.exports = function (string) {
   if (m) return m.slice(1, 4)
 
   // https://docs.gitlab.com/ce/user/group/subgroups/
-  if (~url.host.indexOf('gitlab')) {
+  // https://git.company.com/ce/user/group/subgroups/
+  if (~url.host.indexOf('git')) {
```

Comment:
Is it worth checking for `git` at this point? There are GitLab deployments out there that do not have `git` in their name.

What would happen if we did not check for `git` and just attempted to process the URL as is?

Reply:
I was debating with myself about this one as well. I left it, as I wasn't sure about the motivation behind this condition.

If we both thinks the same, I guess we should remove it.